sspl_test/mock_server

```diff
@@ -18,6 +18,9 @@
 
 import json
 import os
+import logging
+log = logging.getLogger('werkzeug')
+log.setLevel(logging.ERROR)
```

Comment:
In QA framework , we are using the same logger obj defined in service_logging file.
what is the role of this log obj?